app/helpers/sessions_helper.rb

```diff
@@ -3,9 +3,31 @@ def log_in user
     session[:user_id] = user.id
   end
 
+  # Remembers a user in a persistent session.
+  def remember user
+    user.remember
+    cookies.permanent.encrypted[:user_id] = user.id
+    cookies.permanent[:remember_token] = user.remember_token
+  end
+
+  # Forgets a persistent session.
+  def forget user
+    user.forget
+    cookies.delete(:user_id)
```

Comment:
```suggestion
    cookies.delete :user_id
```
chỗ nào bỏ đc ngoặc thì bỏ e nè